src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2059,6 +2063,94 @@ def aks_upgrade(cmd,    # pylint: disable=unused-argument, too-many-return-state
     return sdk_no_wait(no_wait, client.create_or_update, resource_group_name, name, instance)
 
 
+def aks_runcommand(cmd, client, resource_group_name, name, command_string="", command_files=None):
+    colorama.init()
+
+    mc = client.get(resource_group_name, name)
+
+    if not command_string:
+        raise CLIError('Command cannot be empty.')
+
+    request_payload = RunCommandRequest()
+    request_payload.command = command_string
+    request_payload.context = _get_command_context(command_files)
+    if mc.aad_profile is not None and mc.aad_profile.managed:
+        request_payload.cluster_token = _get_dataplane_aad_token(cmd.cli_ctx, "6dae42f8-4368-4678-94ff-3960e28e3630")
```

Comment:
what is this? can we define a const?

Reply:
this is AKS AAD ServerApp ID, for all AAD enabled aks cluster.

Reply:
sure, I can defined this as const.